deep_agent/src/agent/llm.py

```
@@ -183,3 +183,146 @@ def _create_vllm_model(
         )
         raise LLMError(f"Failed to create vLLM model '{model_name}': {e}") from e
+
+
+def _create_openai_model(
+    model_name: str,
+    temperature: float,
+    max_output_tokens: int,
+) -> BaseChatModel:
+    """Create a native OpenAI model."""
+    if not settings.OPENAI_API_KEY:
+        raise ValueError("OPENAI_API_KEY is required for OpenAI models.")
```

Ruff (0.16.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)